aspnetcore/signalr/java-client.md

```diff
@@ -91,12 +91,44 @@ HubConnection hubConnection = HubConnectionBuilder.create("YOUR HUB URL HERE")
     })).build();
 ```
 
+::: moniker range=">= aspnetcore-5.0"
+
+### Passing Class information in Java
+
+When calling the `on`, `invoke`, or `stream` methods of `HubConnection` in the Java client, users should pass a `Type` object rather than a `Class<?>` object to describe each `Object` passed to the method. A `Type` can be acquired using the provided `TypeReference` class. For example, using a custom classes named `Foo` and `Bar`, the following code gets each `Type`:
+
+```java
+Type fooType = new TypeReference<Foo>() { }).getType();
+Type barType = new TypeReference<Bar>() { }).getType();
+```
```

Comment:
This only necessary for generic types, right? I think `Bar.class` would work fine.

Reply:
Oops, yes, you're right